README.md

```diff
@@ -1,2 +1,78 @@
 # server-template
```

Comment:
Isn't it `Spine-based Server Quickstart`, or something like this? We wanted to avoid the word `template`.